docs/requirements.md

```diff
@@ -11,13 +11,27 @@ The next sections describe in detail how to set up these prerequisites.
 
 The cluster running Codacy must satisfy the following requirements:
 
--   The infrastructure hosting the cluster must be provisioned with the hardware requirements described below
+-   The infrastructure hosting the cluster must be provisioned with the hardware and networking requirements described below
 -   The orchestration platform managing the cluster must be one of:
     -   [Kubernetes](https://kubernetes.io/) **version 1.14.\*** or **1.15.\***
     -   [MicroK8s](https://microk8s.io/) **version 1.15**
 -   Tiller, the server part of [Helm](https://v2.helm.sh/) **version 2.16**, must be installed in the cluster
 -   The [NGINX Ingress controller](https://github.com/kubernetes/ingress-nginx) must be installed and correctly set up in the cluster
 
+### Cluster networking requirements
+
+The cluster must be configured to accept and establish connections on the following ports:
+
+|          | Service      | Protocol/Port | Notes                                                        |
+| -------- | ------------ | ------------- | ------------------------------------------------------------ |
+| Inbound  | SSH          | TCP/22        | Optional, to access the infrastructure remotely.             |
+| Inbound  | HTTP         | TCP/80        |                                                              |
+| Inbound  | HTTPS        | TCP/443       |                                                              |
+| Outbound | PostgreSQL   | TCP/5432      | Connection to the PostgreSQL DBMS                            |
+| Outbound | SMTP         | TCP/25        | Connection to your SMTP server                               |
+| Outbound | SMTPS        | TCP/465       | Connection to your SMTP server over TLS/SSL                  |
+| Outbound | Git provider | \*            | Connection to the ports required by your remote Git provider |
```

Comment:
Shouldn't we have dockerhub here too?

Reply:
We should. :+1: 

Unfortunately it seems it's not trivial to define the firewall rules to whitelist access to the Docker Hub. Docker ends up not providing the details (https://success.docker.com/article/is-there-a-whitelist-of-ips-used-by-docker-hub) and I also couldn't find a definitive answer online.

So I suggest that we add another row to the table with the aim of reminding the administrator about this and not so much to actually provide the values needed to configure a firewall.

Reply:
I think its two different problems. We never had ips in the configuration.
They could probably whitelist the dns instead, at least that was what we provided before.

Reply:
I tried to google about it and maybe it was never an option. Not sure why we had it there before.